resources/stubs/requests/ProfileUpdateRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Validation\Rule;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Foundation\Http\FormRequest;
+
+class ProfileUpdateRequest extends FormRequest
+{
+    public function rules()
+    {
+        return [
+            'name' => ['required', 'string', 'max:255'],
+            'email' => ['required', 'email', 'string', 'max:255', Rule::unique('users')->ignore(Auth::user())],
+            'password' => ['sometimes', 'required_with:old_password', 'string', 'confirmed', 'min:8'],
```

Comment:
@krekas I don't think we need `required_with:old_password` here, it may have been copied from another demo project where that old_password field existed.